services/efs-guardian/tests/unit/conftest.py

```diff
@@ -0,0 +1,69 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from pathlib import Path
+from typing import AsyncIterator
+
+import httpx
+import pytest
+from asgi_lifespan import LifespanManager
+from faker import Faker
+from fastapi import FastAPI
+from pytest import MonkeyPatch
+from pytest_simcore.helpers.utils_envs import EnvVarsDict, setenvs_from_dict
+from simcore_service_efs_guardian.core.application import create_app
+from simcore_service_efs_guardian.core.settings import ApplicationSettings
+
+pytest_plugins = [
+    "pytest_simcore.cli_runner",
+    "pytest_simcore.environment_configs",
+    "pytest_simcore.repository_paths",
+]
+
+
+@pytest.fixture(scope="session")
+def project_slug_dir(osparc_simcore_root_dir: Path) -> Path:
+    # fixtures in pytest_simcore.environs
+    service_folder = osparc_simcore_root_dir / "services" / "resource-usage-tracker"
+    assert service_folder.exists()
+    assert any(service_folder.glob("src/simcore_service_resource_usage_tracker"))
+    return service_folder
+
+
+@pytest.fixture
+def app_environment(monkeypatch: MonkeyPatch, faker: Faker) -> EnvVarsDict:
+    envs = setenvs_from_dict(
+        monkeypatch,
+        {
+            "POSTGRES_HOST": faker.domain_name(),
+            "POSTGRES_USER": faker.user_name(),
+            "POSTGRES_PASSWORD": faker.password(),
+            "POSTGRES_DB": faker.pystr(),
+        },
+    )
+
+    return envs
+
+
+@pytest.fixture
+def app_settings(app_environment: EnvVarsDict) -> ApplicationSettings:
+    settings = ApplicationSettings.create_from_envs()
+    return settings
+
+
+@pytest.fixture
+async def initialized_app(app_settings: ApplicationSettings) -> AsyncIterator[FastAPI]:
+    app = create_app(app_settings)
+    async with LifespanManager(app):
```

Comment:
Use instead some approach like this 
https://github.com/ITISFoundation/osparc-simcore/blob/master/services/api-server/tests/unit/conftest.py#L89-L125

it defines two simple fixtures, namely `app` and `client`.

Reply:
Instead of copy/pasting I recommend you start creating `pytest_plugin`s with common fixtures so we unify fastapi-like services as much as possible and make these "skeletons" as small as possible